packages/patternfly-4/react-core/src/components/Page/Page.tsx

```diff
@@ -63,7 +65,8 @@ export class Page extends React.Component<PageProps, PageState> {
     isManagedSidebar: false,
     defaultManagedSidebarIsOpen: true,
     onPageResize: (): void => null,
-    mainContainerId: null as string
+    mainContainerId: null as string,
+    role: undefined as string
```

Comment:
Should this default to `main` rather that undefined?

Reply:
AXE sometimes throws an error when it's set to `main`, so @seanforyou23 and I decided it was better to remove the default and have the user configure it to whatever they prefer, rather than having it be hard-coded.

Reply:
@jessiehuff the core doc say this attribute is required.  Is that not the case,  do the docs need to be updated? https://pf4.patternfly.org/documentation/core/components/page#accessibility

Reply:
@tlabaj I think with this change, the docs would need updated to reflect it. :)

Reply:
@jenny-s51  can you open issue in core please

Reply:
@tlabaj yes!